Makefile

```diff
@@ -13,41 +7,67 @@ ifeq ($(TAGGED_VERSION),)
 endif
 VERSION ?= $(shell echo $(TAGGED_VERSION) | cut -c 2-)
 
-.PHONY: compare-deps
-compare-deps: Gopkg.lock GlooE-Gopkg.lock print-info
-	go run scripts/compare_dependencies.go Gopkg.lock GlooE-Gopkg.lock
+#----------------------------------------------------------------------------------
+# Retrieve GlooE build information
+#----------------------------------------------------------------------------------
+GLOOE_DIR := _glooe
+z := $(shell mkdir -p $(GLOOE_DIR))
+
+.PHONY: get-glooe-info
+get-glooe-info: $(GLOOE_DIR)/Gopkg.lock $(GLOOE_DIR)/verify-plugins-linux-amd64 $(GLOOE_DIR)/build_env
 
-GlooE-Gopkg.lock:
-	curl -o GlooE-Gopkg.lock http://storage.googleapis.com/gloo-ee-dependencies/$(GLOOE_VERSION)/Gopkg.lock
+$(GLOOE_DIR)/Gopkg.lock:
+	curl -o $@ http://storage.googleapis.com/gloo-ee-dependencies/$(GLOOE_VERSION)/Gopkg.lock
+
+$(GLOOE_DIR)/verify-plugins-linux-amd64:
+	curl -o $@ http://storage.googleapis.com/gloo-ee-dependencies/$(GLOOE_VERSION)/verify-plugins-linux-amd64
+
+$(GLOOE_DIR)/build_env:
+	curl -o $@ http://storage.googleapis.com/gloo-ee-dependencies/$(GLOOE_VERSION)/build_env
 
-# TODO: remove
-.PHONY: print-info
-print-info:
-	@echo BUILD_ID: $(BUILD_ID)
-	@echo TAGGED_VERSION: $(TAGGED_VERSION)
-	@echo VERSION: $(VERSION)
-	@echo RELEASE: $(RELEASE)
 
 #----------------------------------------------------------------------------------
-# Build, test and publish example plugins
+# Compare dependencies against GlooE
+#----------------------------------------------------------------------------------
+
+.PHONY: compare-deps
+compare-deps: Gopkg.lock $(GLOOE_DIR)/Gopkg.lock
+	go run scripts/compare_dependencies.go Gopkg.lock $(GLOOE_DIR)/Gopkg.lock
+
+
+#----------------------------------------------------------------------------------
+# Build plugins
 #----------------------------------------------------------------------------------
 EXAMPLES_DIR := examples
 SOURCES := $(shell find . -name "*.go" | grep -v test)
 
-.PHONY: publish-examples
-publish-examples:
+define get_glooe_var
+$(shell grep $(1) $(GLOOE_DIR)/build_env | cut -d '=' -f2)
```

Comment:
feels more natural to use awk here

i.e.,
```
awk -F"=" '{print $2}' $(GLOOE_DIR)/build_env
```
you can combine that with pipe and grep, but awk syntax is expressive enough I believe you can get what you want without grep

Reply:
I believe the syntax is:
```
awk -F"=" '/$(1)/ {print $2}' $(GLOOE_DIR)/build_env
```

Reply:
Not very familiar with `awk`. I tried the command above and it seems to range over all the lines in the file. I did a couple of google searches but could not find a way of matching only one line. I would leave it like this for now, feel free to update if you want.

Reply:
Tried:
```
awk -F"=" '/$(1)/ {print $2}' $(GLOOE_DIR)/build_env
```
but somehow make does not like the $(1) being quoted... I'll leave it like it is for now.